packages/components/card/src/card.tsx

```diff
@@ -25,12 +26,33 @@ export type TCardProps = {
    */
   className?: string;
   children?: ReactNode;
+  /**
+   * The callback function to be executed when the Card component is clicked. Prefer this for managing side effects rather than navigation.
+   */
+  onClick?: () => void;
+  /**
+   * The URL that the Card should point to. If provided, the Card will be rendered as an anchor element.
+   */
+  to?: string;
```

Comment:
```suggestion
  to?: string | LocationDescriptor;
```

Reply:
Can you provide an example of how an object of type `LocationDescriptor` is unpacked for use with `<a>`'s `to` prop? Having some difficulty figuring this out 😅

Reply:
Ah, I see this [here](https://github.com/commercetools/ui-kit/blob/main/packages/components/link/src/link.tsx#L140), just want to make sure this is the direction we want to go in?

Reply:
Yes, that would be great, thanks!

Reply:
See 6f6f96fef23f45736ca82dd50b0386a6c3279f27 for the fix - thank you!